README.md

```diff
@@ -11,6 +11,7 @@ Condé Nast - copilot JavaScript utilities
 ## Maintainers
 
 - Phil Gold ([@pgoldrbx](https://github.com/pgoldrbx) / phil_gold@condenast.com)
+- Zack Tollman ([@tollman](https://github.com/tollmanz) / zackary_tollman@condenast.com)
```

Comment:
Since I'm becoming more familiar with these packages, I might as well be a maintainer. Slipping this update into a "meta" update.